policies/approval.yml

```diff
@@ -52,21 +52,10 @@ approval_rules:
         # will be accepted as approval candidates. Default is true.
         github_review: true
 
-  # TODO: check who has permissions to add labels before enabling
```

Comment:
I confirmed that only users with write access can add labels 👍

Reply:
I hope so, I'd hate to have some random labelling my PRs :)